tests/components/group/test_binary_sensor.py

```diff
@@ -0,0 +1,94 @@
+"""The tests for the Group Binary Sensor platform."""
+from os import path
+
+from homeassistant.components.binary_sensor import DOMAIN as BINARY_SENSOR_DOMAIN
+from homeassistant.components.group import DOMAIN
+from homeassistant.const import ATTR_ENTITY_ID, STATE_OFF, STATE_ON, STATE_UNAVAILABLE
+from homeassistant.helpers import entity_registry as er
+from homeassistant.setup import async_setup_component
+
+
+async def test_default_state(hass):
```

Comment:
This test, or a new one, should verify that the configurable unique_id and attributes (name, device class, icon if added) are working. The same improvement should be done for the other groups in a follow-up.

Reply:
Aren't the tests for the base classes already covering this?

Reply:
I don't think so, since the base class don't support configuring `unique_id` etc.?

Reply:
I added a test for it. They can later be removed from all classes in the package if the attribute is not recommended.